pig/src/main/java/org/jboss/pnc/bacon/pig/impl/config/BuildConfig.java

```diff
@@ -235,8 +235,7 @@ public boolean isTheSameAs(BuildConfiguration old, boolean skipBranchCheck, bool
     }
 
     private boolean urlsEqual(SCMRepository repo) {
-        return StringUtils.equals(externalScmUrl, repo.getExternalUrl())
-                || StringUtils.equals(scmUrl, repo.getInternalUrl());
+        return StringUtils.equals(scmUrl, repo.getInternalUrl());
```

Comment:
`scmUrl` field can also be external right?

Reply:
yup :) you can put external url e.g. github to scmUrl field

Reply:
sure